spec/datetimeformat.html

```diff
@@ -224,35 +225,75 @@
         1. Return FormatDateTime(_dtf_, _x_).
       </emu-alg>
 
+      <emu-note>
+        The function returned by [[Get]] is bound to this DateTimeFormat object so that it can be passed directly to Array.prototype.map or other functions.
+      </emu-note>
+
       <p>
         The *length* property of a DateTime format function is 1.
       </p>
     </emu-clause>
 
-    <emu-clause id="sec-formatdatetime" aoid="FormatDateTime">
-      <h1>FormatDateTime (dateTimeFormat, x)</h1>
+    <emu-clause id="sec-datetime-formatToParts-functions">
+      <h1>DateTime FormatToParts Functions</h1>
 
       <p>
-        When the FormatDateTime abstract operation is called with arguments _dateTimeFormat_ (which must be an object initialized as a DateTimeFormat) and _x_ (which must be a Number value), it returns a String value representing _x_ (interpreted as a time value as specified in ES2015, 20.3.1.1) according to the effective locale and the formatting options of _dateTimeFormat_. This abstract operation functions as follows:
+        A DateTime formatToParts function is an anonymous function that optionally takes an argument _date_. It performs the following steps:
       </p>
 
       <emu-alg>
+        1. Let _dtf_ be the *this* value.
+        1. Assert: Type(_dtf_) is Object and _dtf_ has an [[initializedDateTimeFormat]] internal slot whose value is *true*.
+        1. If _date_ is not provided or is *undefined*, then
+          1. Let _x_ be *%Date_now%*().
+        1. Else,
+          1. Let _x_ be ? ToNumber(_date_).
+        1. Return FormatToPartsDateTime(_dtf_, _x_).
+      </emu-alg>
+
+      <emu-note>
+        The function returned by [[Get]] is bound to this DateTimeFormat object so that it can be passed directly to Array.prototype.map or other functions.
+      </emu-note>
+
+      <p>
+        The value of the [[Set]] attribute is *undefined*.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="alg-CreateDateTimeParts">
+
+      <h1>CreateDateTimeParts(dateTimeFormat, x)</h1>
```

Comment:
In #79 @caridy wanted to use the verb _format_ for a similar abstract operation: `FormatNumberToParts`. Perhaps we could rename this here as well for consistency to `FormatDateTimeToParts'?  This will also be more inline with the name of the existing abstract operation: `FormatDateTime`.